state-chain/pallets/cf-vaults/src/vault_activator.rs

```diff
@@ -40,12 +43,17 @@ impl<T: Config<I>, I: 'static> VaultActivator<<T::Chain as Chain>::ChainCrypto>
 					);
 				},
 			}
-		} else {
-			// No active key means we are bootstrapping the vault.
+			FirstVault::False
+		} else if ChainInitialized::<T, I>::get() {
+			// VaultStartBlockNumbers being empty means we are bootstrapping the vault.
 			PendingVaultActivation::<T, I>::put(
 				VaultActivationStatus::<T, I>::AwaitingActivation { new_public_key },
 			);
 			Self::deposit_event(Event::<T, I>::AwaitingGovernanceActivation { new_public_key });
+			FirstVault::True
+		} else {
+			PendingVaultActivation::<T, I>::put(VaultActivationStatus::<T, I>::Complete);
```

Comment:
To be clear - this is how we are going to bypass the need to activate arb atm? We should add a comment about *why* this code is the way it is, it's definitely not obvious without a lot of context

Reply:
yes, added some comments.